tendrl/common/alert.py

```diff
@@ -0,0 +1,153 @@
+import json
+from tendrl.common.config import TendrlConfig
+from tendrl.common.etcdobj.etcdobj import Server as etcd_server
+from tendrl.common.singleton import to_singleton
+
+
+alert_severity_map = {
+    'INFO': 0,
+    'WARNING': 1,
+    'CRITICAL': 2
+}
+config = TendrlConfig("commons", "/etc/tendrl/tendrl.conf")
+
+
+class Alert(object):
```

Comment:
This needs to defined via Tendrl definitions under say the alerting namepsace or node-agent namespace, we dont use standalone objects